scipy/spatial/_spherical_voronoi.py

```diff
@@ -156,34 +159,38 @@ def __init__(self, points, radius=1, center=None, threshold=1e-06):
         if cKDTree(self.points).query_pairs(self.threshold * self.radius):
             raise ValueError("Duplicate generators present.")
 
-        radii = np.linalg.norm(self.points - self.center, axis=1)
-        max_discrepancy = np.abs(radii - self.radius).max()
+        radii: np.ndarray = np.linalg.norm(self.points - self.center, axis=1)
```

Comment:
I reckon here the annotation was necessary since `np.linalg.norm` isn't typed yet?

Reply:
No, probably not, I went a bit aggressive on just typing all over the place to see if mypy could catch any issues. I should probably scale that back a bit.

Reply:
I've backed out typing here and anywhere else it wasn't actually needed to fix a `mypy` error.